<div id="fileContents"><h2><strong>Frontiers of Biomedical Engineering: </strong>Lecture 4 Transcript </h2><table cellspacing="0" cellpadding="0" id="transcriptHeader" summary="transcript header">	<tbody>		<tr>			<td id="transcriptDate">January 24, 2008</td>			<td id="transcriptNav"><span id="forwardNav"><a href="javascript:history.go(-1);">&lt;&lt; back</a></span></td>		</tr>	</tbody></table></p><p><b>Professor Mark Saltzman: </b>So, we're going to continue talking today about DNA. In particular, we're going to focus today on sort of how to manipulate and use DNA in some applications, and this is a huge area of science and technology. You know this, you can - it's hard to pick up a newspaper or a news magazine without hearing some new application of DNA technology. What I'm going to do is focus on a couple basic things that turn out to be really important for general applications, and then I'll talk not in too much detail about a few applications of DNA that you're probably familiar with to try to give you a framework to hang this on. </p><p>This, I think Chapter 3, describes fairly well the things we'll talk about today. You've probably already noticed that there's some material that's set aside from the text in boxes, and I encourage you to particularly look at those boxes for this chapter. There's one on DNA fingerprinting, for example, that gives you a little bit more detail about that specific technique; another one on production of therapeutic proteins. I'll talk about these both a bit today but I encourage you to read those for more information. </p><p>I want to start where we left off last time. We talked about the structure of DNA, how it works in terms of a physical chemical model of the DNA molecules. We talked about base pairing and how that leads to this process of hybridization or very specific matching between complimentary strands. We talked very superficially about the biological process going from DNA to protein, so the process of transcription, RNA processing, and translation to produce a protein. I ended with this picture that shows you a little bit about control of gene expression. The important concept is that, while every cell in your body has the capability of making all the proteins that are needed throughout your body, not every cell is doing that at any given time. Only certain genes are being expressed and it's the family of genes that are being expressed in a cell, likewise the family that are not being expressed, that determines what a cell is like, how it functions. What's called the phenotype of a cell and we'll talk more about this next week when we start talking about cells and a little bit about cell physiology. </p><p>There are multiple mechanisms that a cell can use to decide which genes it is expressing at any one time and which ones are not expressed. I showed you this in this picture here and those levels of control can be at the level of transcription. There are molecules in cells that give the DNA the signal that it's time to transcribe and express a gene, those are called transcription factors, we'll talk about them a bit later. There's interfering with RNA processing, and I'm going to talk about that in the next couple of slides because there's a couple of new methods for - or potentially interfering with gene expression in living animals that have been developed based on changing - interfering with DNA transcription and the ability of messenger RNA to be translated. You could interfere at these later levels as well, for example, by augmenting RNA degradation. If the messenger RNA for protein is not present in a cell, then that can't be translated, obviously, and the protein can't be made. </p><p>These two new medical therapies that I mentioned are based on interfering with the biology of RNA, and one is older than the other and the older one is called anti-sense therapy. When you think about a gene or a transcript, the messenger RNA copy of a gene, you know that for every sequence of a nucleic acid there's a complimentary sequence. Now of the two complimentary sequences, one of them encodes the gene. One of them has the right sequence of codons to specify the amino acid sequence of the protein, and the other one has a complimentary sequence. You know from our discussion last time that these two complimentary strands are not mirror images of one another, they're not identical, they're complimentary. They face in the opposite direction and you could predict the properties of one from base pairing rules of the other but they're not the same. One of the strands encodes the protein, encodes for the protein, the other does not. The one that does is called the sense strand and the other one is called the anti-sense strand, anti meaning it's the compliment and it will hybridize to the sense. </p><p>What if you knew what the sequence of a gene was? A gene, let's say it's the gene for insulin. I'll use that one as the example because it's a familiar one to most people and you know that the protein insulin is made only in your pancreas and certain cells of the pancreas. So that means those cells are continually making messenger RNA and that messenger RNA is being converted into protein. Well, what if you knew the sequence for the messenger RNA that made insulin and you designed another single stranded DNA or RNA molecule that was the exact opposite, or the exact compliment, I should say, of that strand? So you made somehow an anti-sense polynucleotide to the insulin gene or some fraction of the insulin gene. </p><p>Well, that anti-sense strand is shown here as the red and the cell is naturally making the blue or sense strand of messenger RNA for a particular protein. If somehow you could take your anti-sense molecules that you've made and you could get them into cells, then by this process of hybridization they would naturally form a pair like this. They would naturally hybridize and form a duplex, or a double stranded nucleic acid. When it's double stranded this gene can't be translated, because you have to have the single strand in order for the transfer RNA to bind and for this process of translation to take place. What you could do then, if you could deliver these red colored molecules here is you could stop specifically the expression of this particular gene in these particular cells. </p><p>Now, what are the challenges there? You've got to be able to make this stuff and you've got to be able to make it in large quantities and we'll talk about how to make nucleic acids in large quantities a little bit later in the lecture. You've also got to get it into the cell. It turns out that getting large molecules like this, particularly large charged molecules like nucleic acids, inside of cells is not so easy. We'll talk about that a little bit later as well. In fact, we'll talk about that concept throughout the course because one of the big challenges of making these sort of new biological therapies work in people to treat diseases is getting the right molecules into the right cells, at the right period of time. </p><p>Now I gave you the example of insulin and you probably wouldn't want to stop insulin production. That might not be a good thing to do. What if this is a gene that's causing a cell to be cancerous? It was a gene that was causing a cell to be malignant and to divide without control, for example. Then you could imagine blocking gene expression would be a therapy. </p><p><b>Student: </b>[inaudible]</p><p><b>Professor Mark Saltzman: </b>You wouldn't want to stop insulin, for example. In fact what you might prefer to do is start insulin production and we'll talk about ways to do that in just a minute. These are ways to stop a gene from being expressed, and there turns out there's lot of applications in that, lots of diseases result from the unwanted expression of certain kinds of genes and cancer is probably the best example of that, but there are many. </p><p>A newer version of this that works in a similar way but a different way is called RNA interference, and it turns out that this is a natural mechanism that cells have. It's a mechanism that they have evolved in order to prevent foreign genes from entering a cell and being expressed. You have mechanisms inside your natural mechanisms inside your cell that allow the cells to degrade unwanted RNA sequences. Those mechanisms are called RNA interference. You might have heard about this because it's been quite an active area of science. </p><p>It turns out to activate RNA interference, you deliver double stranded RNA. Certain double stranded RNA sequences will cause in the cell a process of degradation of very specific RNA sequence. This involves mechanisms that are still being understood, but if you've studied some biology or read about this you've heard about the protein complex called Dicer. Dicer is an internal cellular mechanism for degrading RNA's. You might have also heard about the RISC complex, or the RNA silencing complex, and these are the biological mechanisms that are involved here and only shown by orange arrows on this slide. The end result is you can design now very specific double stranded RNA sequences, that when delivered into cells again will activate this process of natural degradation of an existing messenger RNA. Of course, if you degrade the messenger RNA at a rapid rate than you'll stop expression of the cells. </p><p>Now the nice thing about this is that the degradation mechanisms seem to persist for some period of time, beyond the time at which you deliver the double stranded RNA; whereas, obviously, this mechanism here is only going to exist for as long as the anti-sense sequence is present. So this might be a longer lasting, more permanent form of elimination of expression of a particular gene. </p><p>I just wanted to introduce those concepts because you've read about them; we'll be talking more about RNA interference in particular as we go on through the course. The rest of the time I want to talk about expression of genes, of new genes. Taking foreign genes, genes that aren't naturally expressed or might not even exist inside a cell and putting them there and putting them there in a way where they work, and by work meaning the gene gets expressed or translated into a protein. </p><p>I'm going to start by talking about a very specific and interesting form of double stranded DNA called a plasmid and plasmids occur in nature. Plasmids turn out to be one of the most powerful and simplest examples of a vector, what's called a vector for delivering DNA into a cell. Now the challenge is not just to get the DNA that encodes a gene into a cell, the challenge is to get it into the cell in a form where the cell can use it, can express it and make proteins from it. The plasmid has some features which allow it to do that. Now to start with, the plasmid is usually shown in a diagram like this as a circle. It's a double stranded circular piece of DNA, meaning that the 3&#23591;, 5&#23591; ends that hang off are joined back together again to form a continuous loop. Again, these plasmids occur naturally in nature; they were discovered particularly in micro-organisms have plasmids that confer biological properties onto them. </p><p>This particular example of a plasmid has several regions. Now, in your book, there's an example of plasmid where I've given you the exact sequence of nucleotides that makes up the whole double stranded DNA molecule. I just give you one of those, right, because you could write down the other one because you know the other complimentary sequence from base pairing? One of the things about these plasmids that makes them very useful is that their entire base pair sequences is known. So you know everywhere on this picture you could write down exactly what the sequence of nucleotides are that make up this vector. </p><p>This region here of this particular plasmid is called the <i>ori</i> or origin of replication. Remember we talked about how DNA replicates itself and that there are enzymes, DNA polymerase that bind to the double stranded DNA, separate it, denature it locally, and then start the process of replication. One of the properties that you would like a plasmid to have is you'd like for cells to be able to replicate it, to make more copies of it. That way you could deliver a small number of vectors and they could amplify into a large number of vectors. So having a place that the cell knows - where the cell knows how to replicate is important and so plasmids have an origin of replication. </p><p>The blue region here is a gene. It's a gene that's on the plasmid, and this particular gene confers a specific biological property to cells that have the plasmid and can use it properly. The property it gives here is called AMP<sup>R</sup> or resistance to Ampicillin. Ampicillin is an antibiotic. Antibiotics are chemicals, usually small organic molecules that will kill micro-organisms like bacteria. If a cell has a gene that makes it resistant to Ampicillin, that means that that micro-organism can survive being exposed to this normally deadly chemical without dying. This is one of the biological properties, the naturally occurring properties of plasmids, is that they exist in microbial populations and they confer on them resistance from toxins that would ordinarily kill them. So being a micro-organism that got a plasmid that give you resistance to an antibiotic would be a good thing - that gave you resistance to something that naturally killed cells like you in your environment would be a good thing. We're going to use that Ampicillin resistance in technological ways and I'll describe that in a minute. </p><p>The rest of this, this sort of beige part of the molecule here is called the polylinker part. This is where - this is the region of the plasmid where we're going to insert the DNA that we're interested in. We're going to have DNA that we would like to make lots of copies of, or we're going to have DNA that we would like to get expressed in a cell, and we're going to put it in this region that's called the polylinker. How we do that will be clear in a few minutes, but this polylinker as is described down here is a site where you can clone in genes. </p><p>Let's assume that we have this plasmid cloning vector and we have some pieces of DNA that we would like to put into a plasmid that we would like to make copies of. DNA cloning, or any kind of cloning just means 'making copies of'. So the process of cloning DNA is taking a few strands of DNA of a gene that you're interested in and making many copies of them, that's cloning, you like to make identical copies. This vector is going to allow us to do this. </p><p>The first step in the process is to take our plasmid which we've selected, and to insert the gene that we want into it. For a minute just assume that we can do this and I'm going to show you how to do it on the next slide. The first step is to take the DNA fragments that we're interested in and put them into this vector by basically cutting open the double stranded DNA and inserting the gene that we like in the region where we've cut. Then we're going to take the newly formed vectors that now are recombinant, they're combined from at least two different sources. The sources are: one, the plasmid vector that we've picked, and the second is these genes that for some reason we're interested in. They might have come from two completely different places, from two completely different species from different parts of the world, and they're put together in a new way and that's why it's called recombinant DNA. Then we're going to take these plasmid vectors and we're going to somehow put them in contact with cells in such a way that the cells ingest the DNA and they use it. </p><p>In this particular example here we're exposing these plasmids to bacterial cells. That's shown in this diagram as little colonies of bacteria that are growing on a plate. You've probably seen agar plates, if you smear a solution that's contaminated with bacteria on it, then that bacteria will grow on this agar rich medium and you'll get many, many copies of the bacteria that you've smeared at low density onto the plate. That's a way of culturing or propagating bacteria. Well, if you do that under the situation where you've put your plasmid into these micro-organisms then you're going to have little colonies that grow many copies of the bacterial cells. Hopefully each one of those cells is containing one or more of the plasmids that you're interested in and those are being copied as well. So what you get on the plate is many copies of the small number of plasmids that you've put in. </p><p>Now how do you find those colonies on a plate that have the plasmid that you want? Well, that's a trick and there are multiple ways to do that. One way is to allow these bacteria to grow on a plate that is loaded with antibiotics like Ampicillin. If this plate has Ampicillin in it, then the only cells that would be able to grow here are cells that have resistance to Ampicillin. If you selected the cells right than the only ones that have that resistance to Ampicillin are the ones that successfully got your plasmid and are using this Ampicillin resistant gene. You could imagine strategies where you have multiple resistance genes on a plasmid, resistance to Ampicillin, to Penicillin, to Erythromycin for example, and you design strategies for separating out which cells are carrying the plasmid that you're interested in. This process of using a biological event like resistance to Ampicillin in order to pick out the cell population that you're interested in is called selection. If we grew these cells on a plate loaded with Ampicillin and we could select cells that have Ampicillin resistance, and this process of selection and cell culture is very important and we'll talk about it more next week. </p><p>How did we put our gene fragments into this plasmid DNA in order to make multiple copies of it, or to clone the gene? Well, it involves several steps. The first step is we had to be able to take this circular DNA and cut it to create a site for our new gene to be added. That cutting is done by special proteins called restriction enzymes. Restriction enzymes are just a kind of enzyme, enzymes are protein molecules that make a chemical reaction go faster, and the chemical reaction that restriction enzymes do is cutting DNA. They do that in a very special way in that they - restriction enzymes are able to identify a particular sequence of bases in a gene. </p><p>There's whole families of restriction enzymes. There are hundreds, thousands of them known now, and each one has a specific character and one aspects of its character is that it only binds and cuts at a particular sequence of DNA. This particular restriction enzyme here recognizes this sequence, GAATTC. When it sees that sequence in a double stranded DNA it will bind there and it will cut. Now another property of restriction enzymes is that they always cut the DNA in the same way. In this case, this particular restriction enzyme cuts symmetrically like this, but not at the same point. It doesn't cut straight across the double stranded DNA but it cuts in this jagged fashion. That is, it cuts between the G and the A here, and it cuts between the G and the A here. When it cuts it leaves sticky ends or un-base paired single stranded regions on each end of the part its cut and that's just a property of many restriction enzymes; not all, some cut blunt, just right down the middle. Most restriction enzyme also recognize symmetric sequences of DNA, GAATTC for example. If you do the base pairing goes exactly the same sequence backwards down here. That's an example of symmetric sequence and it happens that most restriction enzymes also recognize those spaces. </p><p>If you cut and you open up a segment of DNA then you've left these sticky ends, for example, and these sticky ends are capable of recognizing each other by the process of hybridization. These will naturally want to reform and they'll want to reform to re-establish this base pairing. They could be pasted back together, and the pasting process takes advantage of this natural process of complimentary hybridization. This gives you a biological mechanism for cutting, using restriction enzymes, and then you denature so that it falls apart, and then you renature so that it comes back together. </p><p>Cutting involves enzymes called restriction endonucleases or restriction enzymes, which I've already mentioned and they have names. Restriction enzymes have names, the particular one that does this function here is called <i>EcoRI.</i> The names all look - they're all italicized and they're capital letters and small letters so that they won't be easy for you to understand, but they are - if you know the nomenclature, easy to understand. This restriction enzyme was found in a natural source, it was found in a micro-organism called <i>E. coli.</i> The first three letters of <i>E. Coli</i> are Eco, so <i>Eco</i>. It was found in strain R, a particular strain of <i>E.&nbsp;coli</i>, and it was the first one found, so <i>EcoRI</i> . There's a nomenclature that's evolved for this. </p><p>Now we know so much about these, they've turned out to be so useful in biotechnology. There are whole catalogs that you go to and buy restriction enzymes. You can look up in the catalog. What are the properties of this restriction enzymes? What base sequence does it recognize? How does it cut? What concentration do I need to use to achieve that? So if you have plasmid where you know all the base pairings than you could go through that plasmid and say I want to cut it right here. What restriction will do that? Or you could ask the question, I have this restriction enzyme, at what regions on this plasmid will it cut? </p><p>The pasting back together occurs partly naturally by this process of hybridization, but hybridization only re-establishes the base pairing. You know that these molecules are also linked in another way, by the phosphate bonds that connect the 3&#23591; and the 5&#23591; carbons of adjacent nucleotides. That doesn't heal naturally but can be reformed by other enzymes called ligases, ligases re-established the phosphate bonds. </p><p>How would I put a gene that I'm interested in into a plasmid? Well the first step would be to cut open the plasmid with a particular restriction enzyme, and then what if I take that same restriction enzyme and I cut up the DNA that I'm interested in. If I cut both the plasmid and my DNA of interest with the same restriction enzyme I'm going to end up with the same sticky ends on both molecules. Now if I put them in contact with one another, the plasmid that's been opened and fragments of the DNA - special fragments that I've produced with the same restriction enzyme, they'll have the same sticky ends, they will naturally hybridize with one another. I apply ligase, and I've got the plasmid that I had before but now with my gene, colored green here, inserted. </p><p><b>Student: </b>[inaudible]</p><p><b>Professor Mark Saltzman:</b> That's a really good question because if I open this up, why wouldn't it just reform with itself, why would it want to have this in here? The answer is it will want to reform with itself, and if I have these in solution than how many reform with itself and how many reform with the molecule I'm interested probably depends on the relative concentrations of both in the solution and what conditions I have it at. It's a statistical process. Some are going to reform and some are going to reform with the gene in, and some probably aren't going to reseal at all under the conditions that I've used. Not every plasmid in your test tube is going to have the right gene inserted in the right way. </p><p>One way that you can look for that gene that you want is by making the cut in your plasmid inside of a gene that encodes for some property like resistance to an antibiotic. If this reforms, so the plasmid reforms back to its native state, that resistance will be recovered. So bacteria that get an unloaded plasmid are going to have resistance to antibiotics. If your gene goes in, you've interrupted the gene for antibiotic resistance and those new organisms aren't going to be resistant to antibiotic anymore. So you could use sort of negative selection in order to find the ones that you want. I don't know if that makes sense or not but - </p><p><b>Student: </b>[inaudible]</p><p><b>Professor Mark Saltzman:</b> Some do, but there's an advantage to having the sticky end there and that you can put things back on, but there are also methods to chemically produce a sticky end where you can take blunt ends and you could add specific nucleotides onto one of the DNA chains by either doing chemistry on a 3&#23591; or the 5&#23591; end and create your own sticky end. Sometimes a blunt cut is useful if you want to sort of grow a sticky end of your choice on it. You're starting to see that there's all different ways that one could take advantage of this fairly simple process of cutting and pasting. That's why molecular biology, one of the reasons why it's turned out to be such a powerful tool, because if you can think creatively you can find all different ways to using these very simple principles to recombine molecules, to make unique new DNA sequences. </p><p>Where does the DNA sequence come from? I want to spend a little time talking about that. Say we've got a human gene that we want to make and let's say it's the human gene for insulin that we want to produce now. All the cells in your body have the gene for insulin in them. Only cells in the pancreas, some cells in the pancreas are making insulin. One way I could try to find the gene for human insulin is to take any cells from any of us, skin cells let's say, and I could identify where on the chromosomal DNA that insulin is likely to occur. I could cut that up into fragments, and I could search in these fragments to try to find the one that has the insulin gene on it. </p><p>Now the problem with that is a problem I mentioned before, that most human genes are not just a straight sequence from beginning to end of the protein that you're interested in. There are encoding regions called exons and those are interrupted by non-coding regions called introns. If I cut up just DNA from the chromosome, what's called genomic DNA, then I'm going to have both exons and introns within the fragments that I create. That might be a good way to do it but it's going to be more of a challenge because you might - you're going to have a lot of these non-coding sequences that are in the way. </p><p>An alternate way is to go to the cell that's making the protein that you want. If it's making the protein you want, it must be producing messenger RNA with that gene one it. That messenger RNA that's being used has already gone through the RNA splicing mechanism and so the introns have already been removed. If I could isolate that messenger RNA - messenger RNA is just a copy of the DNA from which it came - so if I could do the process of reverse transcription, that is instead of transcription which goes from DNA to RNA, if I could go backwards from RNA to DNA, I could recover a DNA version of the gene that I'm interested in. </p><p>It turns out that we can do that now because we have an enzyme called reverse transcriptase, which is able to take single stranded messenger RNA and make DNA out of it. Now you've heard about reverse transcriptase someplace before, right? Anybody heard of reverse transcriptase? </p><p><b>Student: </b>[inaudible]</p><p><b>Professor Mark Saltzman: </b> HIV, HIV is a natural virus that contains an enzyme in it. Why does it contain reverse transcriptase in it? Because HIV is an RNA virus and if it enters your cell the only way it can replicate, it can put its DNA into your cells, is by first making DNA out of its RNA genome. We're going to talk more about this later. </p><p>Reverse transcriptase is a naturally occurring protein, it has a biological function in HIV, but we can use it for a technical logical function here by going backwards on the biological path from messenger RNA to DNA. Now DNA that's produced this way is not called genomic DNA because this doesn't match the DNA in your genome, on your chromosomes, right? The introns are gone now. It's called cDNA or complimentary DNA to indicate that it's a copy, a complimentary copy of the messenger RNA. </p><p>This is a much more efficient way to get DNA for a gene that you're interested for a couple of reasons. One is the processing has already been done so the introns are already out, so you don't have to figure out what's exon and what's the intron, it's already done for you. Plus, if you're looking for insulin, if you're looking for the gene for insulin you're going to cells that are making it already, they have abundant messenger RNA so it's much easier to separate out and identify the gene that you're interested in. You're not fishing through a whole chromosome in order to find what you want, but you're going to a cell that's already enriched in it. Does that make sense? </p><p> I want to talk about one other technique and then I'm going to give you some examples about how to use these in the last few minutes. The technique I want to talk about is one called Polymerase Chain Reaction. This is another way to clone or make many copies of a gene of interest. Now one of the big advantages of plasmids, I already mentioned, is that you can take this plasmid and a plasmid is one - you can think of it as a highly tuned machine for copying itself. A plasmid is a highly tuned machine for making copies of itself. You take that plasmid; you put in the micro-organism you're going to make many, many copies of that plasmid. We're cloning a gene of interest to us by sort of hijacking that biological mechanism by putting our gene on this plasmid and so our gene gets copied many times, as the plasmid gets copied many times. That's a way of cloning DNA. Here's another - that's a biological way of cloning it. We're using biological mechanisms and we're growing cells in order to accomplish it. </p><p>Here's a way that you can do it in a test tube without using any cells by this process called Polymerase Chain Reaction. I'll describe how it works here. You have a fragment of some kind of DNA that you would like to clone or make many copies of. Now in general these are small fragments. They're not whole genomes, but they're some small piece of DNA maybe the size of a protein. You put this chromosomal DNA or this DNA that you're interested in, double stranded into a test tube. You do that together with primers and with nucleotides, because if you're going to synthesize DNA you need the raw material of DNA, you need the individual nucleotides. </p><p>You add a special DNA polymerase called <i>Taq</i> polymerase is a polymerase that was identified, a DNA polymerase that was identified from an organism that lives in regions of the earth that are constantly at high temperature. <i>Taq</i> stands for <i>thermos aquaticus</i>, it's a marine micro-organism that lives near these hydrothermal events in the bottom of the sea and they live under very high temperatures and pressures all the time. So their enzymes are tuned, unlike our enzymes, which are tuned for working most efficiently at 37&#176; centigrade, <i>Taq</i> is used to living at 90&#176; centigrade. So its enzymes operate most efficiently at this elevated temperature. </p><p>You take advantage of that, you put it in a test tube, together with nucleotides and primers and your DNA. Now you start a process, a cyclic process, where the first step in the process is denaturing the DNA. You can do that by gently adding heat and making it basic, but you make the two DNA strands separate. When they separate, the primers that you've added automatically bind through the process of hybridization, and then you turn up the temperature to the optimum for <i>Taq</i> polymerase and DNA synthesis starts. The polymerase starts a process of replication of your DNA sequences. You first separated your DNA, let the primers bind, then turn on the enzyme, and it makes copies of each one of these. You then denature again, each one of these strands gets separated, primers bind, turn on the polymerase, a new strand is made. </p><p> If you repeatedly go through this cycle of denaturing, synthesizing, denaturing, synthesizing you'll get many, many copies of DNA. The number of copies you get depends on how many times you do the cycling. If you start with only one, you have two pieces of DNA, then you'll get 2 to the N<sup>th</sup> fragments after N cycles because each cycle you're doubling the number. </p><p>This has turned out to be a very powerful technique because it's relatively rapid. It's totally synthetic, there's no micro-organism involved in making the DNA for you,. You can do it very rapidly in a laboratory, and fairly expensively now. It's really been a powerful tool in molecular biology because I might have a very small copy of a gene that I'm interested in and I can make enough copies that I can start to do something else with it. You can also use PCR to identify specific genes that are present in a biological sample. You can use PCR for fingerprinting as well, for looking in some unknown fluid for example, is a gene that I'm interested in there? You could use PCR to do that because if you amplify a gene in PCR, that gene had to be present from the beginning. </p><p>Well let's talk about some mechanisms for using this and I'll start with a simple example of how one can detect a gene in a fluid, in a blood sample for example, where that gene is unknown. This takes advantage of the very specific properties of restriction enzymes. This particular example involves the gene for sickle cell anemia. Sickle cell anemia is a disease that affects a subset of the population. It's a single gene defect and we know exactly where the genetic defect is in sickle cell. It's on the gene for hemoglobin and the hemoglobin that you produce if you have sickle cell anemia is not quite right because there's one base difference. If you have normal hemoglobin you have this sequence here, CCTG<b>A</b>GGAG. In sickle hemoglobin you have a sequence that's only slightly different within the gene, CCTG<b>T</b>GGAG, so there's a thymine that's substituted for an alanine and that results in making hemoglobin that's not quite right that doesn't function in the same way as the normal hemoglobin gene does. </p><p>How could I identify in a chromosomal DNA sample whether the sickle gene is present? Well one way you could do it is by saying 'if I have this one base pair difference then this sequence is going to be cut by a specific restriction enzyme that recognizes the sequence CTGAGGA'. That restriction enzyme, there is a restriction enzyme that does that and it's MST-3. If I took this same restriction enzyme and tried to cut the chromosomal DNA of a sickle patient, it wouldn't cut at that point because the wrong sequence is there. Here's - how can I find that, right? Here's the difference between normal hemoglobin and sickle hemoglobin, I can make - I can see a chemical difference, I can exploit it, how can I find it? </p><p>Well the way to find it is by using a process called electrophoresis and Southern blotting. Electrophoresis is described in the chapter here, I'll just describe it briefly. If you have DNA fragments, so this is DNA that you've cut up into fragments using restriction enzymes for example. You load them into the top of a gel and a gel is - in this case it's synthetic polymer gel but it looks very much like Jell-O. I'm sure some of you have dealt with electrophoresis gels in the past and it's really just a slab of something like Jell-O where you can put a sample at one end of the Jell-O, your sample of fragmented DNA. Now you apply electrical charge across that Jell-O. Because it's a gel electricity is going to move through the gel because there are ions in it the same way - for the same reason you don't drop an electrical device into the bathtub because charge moves through water that has ions in it. </p><p>Charge is going to move through this gel and DNA is charged. It's loaded with phosphates which are negatively charged. They're going to move from a negative pole to a positive pole, so they're going to move through this gel. Well if you've created fragments here and they're fragments of different size, the small ones are going to move faster than the big ones. So the DNA is going to get spread out on this gel according to size, with the small ones going farther and the large ones not going as far. </p><p>If I can run the gel for some period of time, run the electrical field, spread it out. Now if I can stain in some way, if I can somehow label the DNA fragments that I'm interested in, I could find out where those fragments are on this gel. I would be able to predict their size, because how far they moved depended on how big they were. How would I label this DNA? The best way to label this DNA is by designing probes or labels that hybridize with specific sequences that you're interested in. This label is DNA that might be made radioactive or made fluorescent, and it has a base pair sequence that is from some other region of the gene that you're interested in. It's only going to bind to fragments that contain that piece of the gene, and it will make those visible to you in some way so you can see where your gene traveled. </p><p>Now something similar to this is the basis of DNA fingerprinting, that's described in one of the boxes in your book. In the specific example, what are we going to do? We're going to take a chromosomal DNA, we're going to digest it with this restriction enzyme, we're going to put it in this tube and run it on a gel, and we're going to see what results down here. In a normal cell DNA that DNA in the normal cell - this DNA gets cut by the restriction enzyme, so the sickle gene ends up - so that hemoglobin gene ends up in two pieces. One that's .2 kilobases long and one that's 1.1 kilobases long - kilobase is 1,000 bases. When I look for it here I'm going to see two pieces, one that's nearly the same length and one that's much shorter. </p><p>If this was a sickle patient, so they had this gene instead it wouldn't get cut and when I went to look for that presence of that gene on this gel, it would appear as one large segment instead of a large one and a smaller one. The absence of this smaller region tells you that this sample came from a sickle patient. You can imagine other ways of doing this, or ways of doing this same thing in different ways. It takes advantage of the specificity of the restriction enzymes, the fact that we know what the gene sequence that we're looking for, and using this technological process of electrophoresis to identify changes that we predict. </p><p><b>Student: </b>[inaudible]</p><p><b>Professor Mark Saltzman:</b> Yeah, this is just one particular example of how to do it, but you could - so you have to identify something that's unique about it and then design a method for identifying that unique thing. Here the unique thing was that there's a restriction site inside that is present in normal DNA and not present in sickle DNA. If you could identify some other change you could do it as well. </p><p>Another example of using this technique is to produce therapeutic proteins from cloned DNA and I'm going to describe this one. It'll probably be the last example I have time for and so I'll go through the rest of them quickly in the next lecture. But here, for example, the idea is to make many copies of a protein for use as a pharmaceutical. Here's an example where we'd like to make insulin, or having the insulin gene would be useful, but if we could take the insulin gene and make many copies of the insulin protein that would be a very useful thing. It turns out that there's lots of proteins that have value as therapeutics and there's a list of them here and some of them you'll recognize. Erythropoietin, commonly called Epo and its function is to treat anemia because it stimulates blood cell production. You might think that its function is to make you more - a better cyclist but that's not its only function. It's used to treat people that have - you've heard about Epo and blood doping, but it's a therapeutic protein as well and very useful for patients with anemia. </p><p> How do you make many copies of cloned DNA? Well one way is to do just exactly what we talked about before. Take a plasmid, cut it open, insert a gene that we want into the plasmid, and then put that plasmid in a host cell, and let the host replicate it. Now in this case the host - you want the host not only to replicate all the DNA, you want it also to express the gene. You're not just trying to clone the DNA, you're trying to also clone or make many copies of the protein. So that's a slightly different thing, right? We want not only for the cell to be able to synthesize the DNA, you want it to be able to express the protein off the DNA as well. That requires design at a different level, not only does it have to have an origin of replication which works so that you can synthesize the DNA, the gene has to be inserted together with a promoter and that promoter is a sequence, a DNA sequence that the host will recognize as a signal to transcribe and translate this protein. </p><p>We'll talk more about promoters as they go on, for now let's just assume that we have a promoter that works in this particular cell. The promoter in this case is called the <i>lac</i> promoter and normally in micro-organisms that <i>lac</i> promoter is used to produce a gene called <i>lacZ</i> which makes a protein called beta-galactosidase. What we're going to do is take this plasmid, with its promoter gene construct in it and engineer it, to take out the <i>lacZ</i> gene, leaving in the promoter and putting in the protein that we want - the gene for the protein that we want. In this case, it says it's GCSF, but it could also be insulin, for example. </p><p>Now when we put insulin in this place behind the right promoter, the cell thinks that it still has the <i>lacZ</i> gene present which it needs for its metabolism. So it's going to express this gene, but you've put a foreign gene in the place of the natural gene, and so the foreign gene gets expressed instead. In this case, GCSF, or it could be insulin and now if I take this cell and I grow it under the right conditions, that is, under conditions in which the cell would normally express the <i>lacZ</i> gene, it's going to express our gene instead and presumably, hopefully, make large quantities of it. </p><p>This was a process that was perfected on an industrial scale by the company Genentech in California, in the 1970s. The first therapeutic protein that was produced was insulin to treat diabetes. Before that, diabetics had been treated with insulin that came from a different source, usually from pigs, so by harvesting and purifying insulin from pig pancreas. Now most diabetics take human insulin made, not in humans, but made in micro-organisms that are growing in a manufacturing facility. </p><p>That is a way of putting a cloned gene into a micro-organism and than using that micro-organism as sort of a factory for the protein. You could do the same thing in mammals and here's one quick example of that, using a different kind of a vector, but inserting a gene into the fertilized egg of a mammal, in this case, a sheep. The biology of this is more complicated than what I described before, but you can inject this DNA into a fertilized egg of a sheep and then implant that fertilized egg with the foreign DNA that's been micro-injected into it, into a foster mother. If that leads to the birth of a young sheep, hopefully that sheep has this gene encoded in its genome now, and it will express the protein that you wanted. </p><p>One example of a way people have used this, is that they've taken the gene that ordinarily produces a milk protein. They put a gene like the insulin gene in place of the milk protein, but behind the promoter that is used to produce milk proteins. So this gene is going to be turned on in the sheep when it makes milk, under conditions where it makes milk. If the sheep grows up in the right way, and people have shown that this will work, you can milk the sheep, the sheep has milk, the milk has insulin in it. Here's an example of using a large animal by inserting the gene that you're interested in, you can make this animal into a factory for the kind of protein that you would like to make. </p>
<p>I'll finish talking about these examples at the beginning of lecture on Tuesday. I'll see you this afternoon in section.</p>
<p></p><p>[end of transcript]</p><p></h2>
  </p>
</div>
<p><a id="backToTop" href="#top">back to top</a></p>