<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1" /><title>Principles of Evolution, Ecology and Behavior</title><body><div id="fileContents">
  <h2><strong> Principles of Evolution, Ecology and Behavior:</strong> Lecture 6 Transcript</h2>
  <table cellspacing="0" cellpadding="0" id="transcriptHeader" summary="transcript header">	<tbody>		<tr>			<td id="transcriptDate">January 26, 2009</td>			<td id="transcriptNav"><span id="forwardNav"><a href="javascript:history.go(-1);">&lt;&lt; back</a></span></td>		</tr>	</tbody></table></p><p><b>Professor Stephen Stearns:</b> Okay, today we're going to talk about the origin and maintenance of genetic variations; and this is continuing our discussion of central themes in the mechanisms of microevolution. The reason we're interested in this is that there cannot be a response to natural selection, and there cannot be any history recorded by drift, unless there's genetic variation in the population. So we need to understand where it, where it, comes from, and whether or not it sticks around. </p><p>If it happened to be the case that every time a new mutation popped up it was immediately eliminated, either for reasons that were random or selective, evolution couldn't occur. If a lot of variation came into the population, and then persisted for a tremendously long time without any sorting, we would see patterns on the face of the earth that are totally different from what we see today. So these issues are actually central issues in the basic part of evolutionary genetics that makes a difference to evolution. </p><p>So the context basically is this. Since evolution is based on genetic change, we need to know where genetic differences come from; and the rate of evolution depends on the amount of genetic variation that's available in the population, so we need to know what maintains the variation. If you were to go back fifty, sixty years, which is what we now think of as the classical view--remember the classical view is a moving window in time--at that point it was thought there wasn't very much genetic variation out there and that evolution was actually limited by the rate at which genetic variation was created. </p><p>Since 1965, with the discovery of protein isozymes, and especially now, since the discovery of ways to sequence DNA very cheaply, we know that's not true. There is a tremendous amount of genetic variation in Nature, and I'm going to show you some of it this morning. So since about 1975, 1980, due to a series of studies, some of them on the Galapagos finches, some of them on the guppies in Trinidad, some of them on mosquitofish in Hawaii, some of them on the world's fish populations responding to being fished, we know that evolution can be very fast when there's strong selection acting on large populations that have lots of genetic variation. </p><p>So really the rate of evolution--and, for example, the issue of climate change and global warming--will all the species on earth be able to adapt fast enough to get--to persist in the face of anthropogenic change on the planet?--that issue is directly addressed by the things we're talking about this morning. </p><p>If there isn't enough genetic change to adapt, say, the grassland populations of the world, or things that are living on mountains, to the kinds of climatic changes that they are going to be encountering, and currently are encountering, they'll go extinct. Ei--they have to either move to a place which is like the one they're in, or they have to adapt to the changed conditions that they're encountering. </p><p>So the outline of the lecture today basically is this. Mutations are the ultimate origin of all genetic variation. Recombination has a huge impact on variation. So what that means basically is that sexual populations have the potential to be much more variable than asexual populations--there is lots of genetic variation in natural populations. And then we will run through four mechanisms that can maintain variations in single genes, and briefly mention the maintenance of variation in quantitative traits. </p><p>So mutations are where these genetic differences come from, and they can be changes in the DNA sequence or changes in the chromosomes, and in the chromosomes they can be changes in how many chromosomes there are in the form of chromosomes or in aspects of chromosome structure. So there can be gene duplications and so forth. Most of the mutations that occur naturally are mutations that are occurring during DNA replication. </p><p>For those of you who are thinking of being doctors, this is important because the probability that a cancer will emerge in a tissue is directly proportional to the number of times cells divide in that tissue; which is why cancers of epithelial cells are much more common than cancers of cells that do not divide. You never get a cancer in your heart muscle, and you frequently get cancers on your skin, and in your lungs, and in the lining of your gut, and that's because every mitotic event is a potential mutation event. </p><p>The kinds of DNA sequence mutations are point mutations; there can be duplications, and in the chromosomes as well there can be inversions and transplacements that go on. Genes can be moved around from one chromosome to another. They can actually be turned around so that they are in the opposite reading direction, along the chromosome. All those things are going on. </p><p>There's good reason to think that an intermediate mutation rate is optimal. If the mutation rate is too low, then the descendants of that gene cannot adapt to changed conditions. If it's too high, then all the accumulation of information on what has worked in the past will be destroyed by mutation; which is what happens to pseudogenes that are not expressed. So some intermediate rate is probably optimal. </p><p>Now a gene that controls the mutation rate will evolve much more easily in an asexual organism than in a sexual species because sexual recombination uncouples the gene for the benefits of the process. Let me illustrate that. </p><p>Suppose that I am engaged in a process that Greg wants to control, and we've got a certain period of time we can do it in, and so he decides that he's going to do it, with me, on a bus going to New York. We go down to the bus station and, because of recombination, he gets into one bus and I get into another. He loses his opportunity to control me, simply because I am now riding in a different bus. </p><p>That's the effect of recombination on genes. Recombination, instead of keeping me on the same chromosome that Greg and I were on, will actually end up putting me into a different body. Okay? So in a sexual organism the gene that's controlling the mutation rate becomes disassociated from the genes whose mutations it might try to control, and therefore even though down in my ride to New York I invent some kind of great process that would benefit Greg, he is now dissociated from it and he doesn't get to benefit from my adaptations. </p><p>So it is much more plausible that we will see genes that are controlling mutation rates evolving in organisms like bacteria and viruses than it is that we will see mutations that control mutation rates evolving in us. There is some reason to think that there is weak selection on them, but it's not as strong as it is in bacteria. And in fact, interestingly, in bacteria you can do experimental evolution and show that the mutation rate will evolve up or down, depending on the circumstances that you put the bacteria under. </p><p>These are some representative mutation rates, and it's good to have some general framework to think about--how frequent is a mutation? So the per nucleotide mutation rate in RNA is about 10<sup>-5</sup>; in DNA it's 10<sup>-9</sup>. So if you start evolving in an RNA world, and you want to lower the mutation rate because your information is getting eroded and you can somehow manage to engineer DNA as your molecule rather than RNA, you can see that you would be able to pick up four orders of magnitude by doing so. That's just because DNA is more stable. </p><p>DNA is a remarkably stable molecule. It's possible to recover DNA from fossil bones. Svante Paabo is in the middle of a project to sequence Neanderthal's genome. He's already got significant chunks of Neanderthal sequence. So DNA is just a remarkably stable molecule. The per gene rate of mutation in DNA is about one in a million; so this is like per meiosis. The per trait mutation rate is about 10<sup>-3</sup> to 10<sup>-5</sup>. The rate per prokaryotic genome is about 10<sup>-3</sup>, and per eukaryotic genome it's between .1 and 10. </p><p>I once saw a really great talk by a guy named Drake, Frank Drake, from NIH--this was like at a big international meeting--Drake walks up to the blackboard and he writes 10<sup>-3</sup> on the blackboard; he's going to give a talk about mutation rates in prokaryotes. He talks for 45 minutes about this number; no PowerPoints, nothing else, he's just speaking very animatedly about how it was that just about all viruses and bacteria appear to have converged on roughly this per generation mutation rate, per genome, which is pretty strong evidence that it's an optimal rate; thousands of species have converged on this rate. </p><p>And I asked him how it was that he gave this great talk without any slides, and he said that he had lost them in the airplane, and that had happened about ten times before, and it was such a great talk without the slides that he just switched completely. So a couple of years ago, actually early last year in this course, I tried giving talks without the PowerPoints. Ninety percent of the class didn't like it and it ten percent of the class did. So that's why you're still getting PowerPoints. Okay? </p><p>Now what is your mutation rate? Well each of you has about four mutations in you that your--new things, your parents didn't have, and about 1.6 of those are deleterious. So this is something that's always going on. And there are about 100 of us in the room; that means there are somewhere around 150 new, deleterious mutations, unique in this generation, sitting here in the classroom. </p><p>Where did they happen? Well they happened fifty times more in males than in females. And there are good biological reasons why. There are many more cell divisions between the formation of a zygote and the production of a sperm than there are between the formation of a zygote and the production of an egg. In human development, and in mammal development, egg production pretty much stops in the third month of embryonic development, at which point all the women in this room had about seven million eggs in their ovaries. </p><p>Since then oocytic atresia, which means the killing of oocytes, has reduced the number of eggs in your ovaries down by nearly seven million. When you began menstruating you had about 1500 eggs in your ovaries. You've gone from seven million down to 1500. When you were born you had gone from seven million down to one million; you'd lost six million of them before you were even born. It appears to be a quality control mechanisms, ensuring that the oocytes that survive are genetically in really good shape. </p><p>So there are very, very different kinds of biology affecting the production of eggs and sperm; females have a mutation screen that males do not. Well the result of that is that there are more mutations in the sperm of older males; they've lived a longer time. Anybody that wants to get in to mate choice and what kinds of reproductive strategies should result from this simple fact is welcome to write a paper on it; there's literature out there. Okay? Not very PC, but it's very biological. </p><p>Okay, recombination. What does recombination do to this mutational variation that builds up in populations? Suppose we had ten genes, and each of those genes had two alleles, and each of those was on a different chromosome. That would mean that just looking at those ten genes, on those ten chromosomes, we could get 3<sup>10</sup> different zygotes. Can anybody tell me why? </p>
  <p><b>Student:</b> [Inaudible]</p><p><b>Professor Stephen Stearns:</b> How many genotypes are there for the first gene? How many different combinations of Aa are there? Three: AA, Aa, aa. So there's three things that the first gene can do. There are three things that the second gene can do. There are three things that the third gene can do. And there are ten genes. So we multiply them to get the number of different combinations, and if they are independently sorting on different chromosomes, that will result in 59,000 different zygotes. </p><p>Now if we had a real eukaryotic genome that had free recombination--which we don't have--and unlimited crossing over--which we don't have--then the number of possible zygotes is about 3<sup>15,000</sup> or 3<sup>50,000</sup>, somewhere along that, that order of magnitude. Well the number of fundamental particles in the universe is only 10<sup>131</sup>. We're talking about numbers which are just inconceivably large. That means that in the entire course of evolution the number of genetic possibilities that are present, just sitting in you, have never been realized. There is a huge portion of genetic space that remains unexplored, simply because there hasn't been enough time on the planet for that many organisms to have lived. </p><p>Now, how--you can see that this would be free recombination with independent assortment of chromosomes. That makes it easier than if it's crossing over, because crossing over happens more frequently the farther genes are apart on a chromosome, and it doesn't happen very often when they're close together. So there's been an evolution of the chromosome number of a lot of species. </p><p>And I've previously told you about ascaris. Ascaris is a nematode that lives in the gut of vertebrates. There is an ascaris that lives in dogs, there's an ascaris that lives in us, and it just has one chromosome. So that's kind of one limit, things with one chromosome. There are species that have hundreds of chromosomes. Sugarcane has I think about 110 chromosomes, something like that. </p><p>So the chromosome number of the species itself evolves, and it can evolve fairly dynamically. There are actually some populations within a single species that have a different chromosome number than other populations within that species, and when individuals from those two populations meet and mate with each other, the offspring often run into developmental difficulties because of this difference in chromosome number. There is such a, uh, contrast in house mice in Denmark. There's a spot where there's sort of a hybrid zone in Denmark, and the house mice on one side of the hybrid zone have difficulty--uh, they're in the same species, but they just have different chromosome numbers--and they have difficulty dealing with the house mice on the other side of that hybrid zone. </p><p>The difference in chromosome numbers appears to have arisen in the house mice during the last glaciation, and they recolonized northern Europe from different places. Some of them came up from Spain. Some of them came up from Greece. They got together in Denmark and they ran into problems. </p><p>Okay, now crossing over also generates a lot of genetic diversity. And the amount of crossing over can be adjusted. Inversions will block crossing over. You take a chunk of chromosome and flip it around, so that in the middle of the chromosome the gene sequences are reversed, and in that section of the chromosome the inversion causes mechanical difficulties. It actually changes the shape of the chromosomes when they line up next to each other, and it inhibits crossing over during meiosis. </p><p>This is one way of taking a bunch of genes that happen to have really helpful interactions with each other, and locking them up in a combination, so that they don't recombine. That has happened, and it's thought to be important in the evolution of quite a few insects, for example. </p><p>Now we can play the mental game of asking ourselves what would happen in a sexual population if we just shut off mutation? We can't actually do it, of course. But how long would it take before we would even notice that evolution had been shut off, if we were just observing the rate at which that population was evolving? </p><p>And the answer to that is kind of interesting. We could wave a magic wand over a moderately large sexual population, completely shut off mutation, and the impact of recombination on the standing genetic diversity in that population would create so many new diverse combinations of genes that it would take about 1000 generations before we would even notice that mutation has been shut off. </p><p>So think back to the beginning of the lecture. I said mutation is the origin of all genetic diversity; and that's true. But once mutation and evolution have been going on for awhile, so much genetic diversity builds up in populations that you can actually shut off mutation and mutation--and evolution will keep going for quite a while. After 1000 generations it'll run out of steam and stop; but it takes quite awhile. </p><p>Okay, so where genetic came--where genetic variation came from and how much there was, was a huge issue and caused a lot of research and controversy for about fifty years. Before 1965, there was the concept of a wild type out there. After 1965--so there was one really good genome, and then there were a few mutations. </p><p>After 1965, with electrophoresis, the impact of Clement Markert's work, and Dick Lewontin, and his colleague Hubby, we've recognized that there's a lot of molecular variation. This concept that each species has a certain genomic type is no longer tenable. There's just a tremendous number of different kinds of genomes out there. Since 1995, we've had a lot of DNA sequence variation and now we've got genomics. </p><p>So I want to illustrate the impact of genomics with something that's just become possible in about the last four years. The HapMap Project was done after the human genome was sequenced, and the motivation of it was to try to associate diseases with common genetic variants. By the way, the upshot of that effort is genes don't normally account for very much, usually about two or three percent of the variation; but that's another story. </p><p>So basically once we had the human genome, it was clear that we could then look for places in genomes that had single nucleotides, that were different, between one person and another; these are called single nucleotide polymorphisms. And to do this the HapMap Project looked at regions of the human genome that were about 10,500 kilobytes long, for 269 individuals. So that's 10,500,000 bases, for each of 269 individuals. And they did it on people from Nigeria, Utah, Beijing and Tokyo. And they discovered that our genome is arranged in blocks. </p><p>There are, within each block, within each let's say rarely recombining section of DNA, there are about 30 to 70 single nucleotide polymorphisms, and that means that you could design a genechip just to pick up enough of these to tag a person as having that particular block of DNA. Okay? So now there are these genechips, and we've discovered that there are some SNPs that are associated with disease. We can see that there are portions of the genome that show signatures of recent selection. This is an interesting literature. </p><p>This is what a little section of our chromosome 19 looks like. Okay? So this is the position along the chromosome, starting at 40,000,000, and going up to 50,000,000 base pairs. The little black dots are all the genes that are in this section of the chromosome, and using the single nucleotide polymorphisms, you can identify people as having a segment of DNA that is not recombining very frequently. And you will notice that they are actually lined up right over places where the recombination rate is pretty high. So you can see breaks in this upper diagram here, showing places where the recombination rate is pretty high. </p><p>So remember, this was done over the entire genome, all of our 23 chromosomes. I am only showing you one tiny part of one chromosome here, and there are actually 650,000 of those blocks that have been identified now in our genome. </p><p>So three years later a group then goes out and takes 928 people, from 51 populations, and looks at how much haplotype diversity there is. Remember, a haplotype is a block that's got some specific nucleotide polymorphisms on it. The Y axis here has 650,000 entries on it. Of course they all blend together, it's hard to see them. The X axis has 928 people arranged across it. This is a sample of human genetic diversity on the planet. You can see there's quite a bit. You can see different colors. Okay? </p><p>Now if you take this and you then use the tools of phylogenetic analysis to ask what kind of historical structure is there in this data set, this is what you get. You get a group in Africa. You can see the emergence of mankind from Africa--this is thought to have happened about 100,000 years ago--and then you get a very, very nice genetic trace of our expansion across the globe. </p><p>We paused for awhile in the Middle East, before we broke out. We were in the Middle East up until about 50,000 years ago, and then there was a group that went into Europe, and other groups then split off from that and set off into Asia. And about probably 40,000 years ago people went to Papua New Guinea and Australia, and probably somewhere around between say 15 and 20,000 years ago, a group of people headed off over the Bering Straight for North America, to become Native Americans, and then another group diversified in East Asia. So there is a huge amount of information in the history of genetic variation. </p><p>So what I'd now like to do is give you four general reasons why this much genetic variation could be maintained in any population. If you look in the textbook you will see that there is also a tremendous amount of genetic variation in wild populations of practically any species, just as there is in humans. In humans it happens to be better analyzed than in almost any other species. But something like that can be done for any species on earth now, and it's getting cheaper and cheaper and cheaper to do so. </p><p>So selection and drift can both explain the maintenance of genetic variation. And for a long time there was a fight within evolutionary genetics about whether what we saw was being explained by selection or drift. It appears not to be a productive question. It's extremely difficult to answer, in any specific case, whether the pattern you see is because of a history of natural selection or because of a history of drift. Both of them are capable of generating quite a few patterns, and those patterns overlap. </p><p>So if you take a very specific case and study it in detail, you can give a leading role to selection or to drift. For example, you can find a signature of selection in a portion of a human chromosome, indicating that there's a gene there that perhaps was affected by a specific disease; that's been done. But the general answer, for all species across the planet, about whether selection or drift is more important, probably is unrealistic. It's probably not a fruitful research effort to try to answer this question. </p><p>So here are the situations that can maintain genetic variation in principle; there are four of them. There can be a balance between mutation and drift; a balance between mutation and selection; there can be heterosis or over-dominance; and there can be negative frequency dependence. So I'm going to step through these now and give you some feeling for how the thinking works on each of them. In so doing, we're going to be dealing with equilibria, and really there are other ways of approaching the analysis, but the equilibrium approach is the one that allows you do it with simple algebra, rather than with complicated computer models. We do it for mathematical convenience. </p><p>We do it also because the periods during which things are in balance may be pretty long, compared to those in which they're dynamically changing--that does appear to be a message of evolution--but with respect to this particular question of the maintenance of genetic variation, we don't really know too much about those periods. Selection can go back and forth; populations can appear to be in stasis when things are going on inside of them. This question is really unresolved. </p><p>We do know that in terms of our immune genes that we share certain polymorphisms with chimpanzees. Those appear to have been things that evolved in terms of disease resistance before humans and chimps speciated, about five to six million years ago. So certainly that genetic variation is five to six million years old. We don't have too many cases where we know that, but there may be many more out there, just undiscovered. </p><p>A little terminology. The fixation probability of a mutation is the probability that it will spread and be fixed in the population. That's equal to its frequency, at any point in time. The fixation time is how long it takes to become fixed in generations. And I put these ideas up on the board earlier, and I'd like to go back to that, because I'd like to have reference to it in a minute. </p><p>So if this is frequency here, it can go from 0 to 1, on the Y axis, and if this is time, over here, this can be many thousands of generations. And the fate of most neutral alleles, when they come into the population, will be to increase in frequency for a little while and then drift out. They have low probability of being fixed because when they first originate they're very rare, and the probability of eventual fixation is just directly equal to their frequency. So in a big population most mutations disappear. But every once in awhile one will drift through, and when it reaches frequency 1.0, it's fixed. Okay? </p><p>So the fixation probability is the probability that out of all of the mutations that might arise, most of which drift out, this one will be fixed; and that's a small number. And the fixation time, how long it takes to be fixed, is on average how long it takes for this process to occur. So that's the fixation time, and that's an average of many such events. So this picture that you're looking at on the blackboard is really just supposed to be an evocative picture, not some kind of precise, concrete state. Because it's representing many, many different genes, they are occurring at all the different possible places in the genome. </p><p>Now for a neutral allele, like the one that I've been sketching there, the fixation rate is just equal to the mutation rate. That doesn't depend on population size. The probability of fixation, as I said, is equal to the current frequency. For a new mutation, one of these guys down here, right at the beginning, that's 1/2N, to be fixed, and 1-1/2N to be lost. That means that most of them are lost. N is the population size. N is a big number. </p><p>Because there are 2N copies of the gene in the population, and if mu is mutation rate, that means in each generation there are 2mu new mutations, and for each of them the probability of fixation is 1/2N. So the rate of fixation of new mutations is about 2mu times 1/2N, which is equal to the mutation rate. That's about 10<sup>-5</sup> to 10<sup>-6</sup> per gene, and that means the molecular clock is ticking once every 100,000 to once every 1,000,000 generations per neutral gene. </p><p>The fixation rate doesn't depend on the population size, and that's because the probability that a mutation will occur in a population depends upon how many organisms are there. You can think of all of their genomes out there as being a net spread out to catch mutations--the bigger then net, the more the mutations are in any given generation--and that will just exactly compensate for the fact that it takes them longer to get fixed. The bigger the population, the longer this process takes. But the bigger the population, the more of these are actually moving through to fixation. Those two things exactly compensate. Okay? </p><p>In a small population most of them are lost. The few that do reach fixation, reach it rapidly, and in large populations more new mutations are fixed, but each one does it more slowly. Those things compensate, and the fixation rate doesn't depend on population size, if you're looking at the whole genome. The number of differences fixed over the whole genome doesn't depend on the size of the population. </p><p>Now there is a technical concept in evolutionary genetics called <i>effective population size</i>, and that is the size of a random mating population, that is not changing in time, whose genetic dynamic would match those of the real one under consideration. And so we know that there are lots of violations of these assumptions. Okay? Populations don't have random mating. They are changing in time; ta-da ta-da. How do we take a real population and then transform it into something that's really easy to calculate? </p><p>Well, there are methods of doing so. The factors that will have to come into consideration are variation in family size, inbreeding, variation in population size, and variation in the number of each sex that is breeding. And so just to illustrate one of these, to give you some idea of its impact, look at cattle in North America. </p><p>There are about 100,000,000 female cattle in North America. They are fertilized by four males, on average, through artificial insemination. So there are four bulls that are inseminating 100,000,000 cows. Genetically speaking, how big is the population? It's just about 16. Okay? So by restricting one sex to a very small number, we have restricted one pathway that the genes can go through to get to the next generation. And by making the male side of it so small, we have biased the probability that a gene will get fixed according to some process like this. </p><p>That male side is a really small population. So it completely outweighs the fact that there are 100,000,000 females there. Because if you think about it, every time one of those genes goes through a female and goes into a baby and grows up the next generation, it's going to go back through the male side of the population--right?--as you go through the generations. And these formulas that have been developed give us the opportunity to take that complex situation and make a quick, useful, back of the envelope calculation of how we can expect genetic drift to be going on in cattle in North America. Basically they are a small population. </p><p>So that's the basis of a mutation-drift balance. The amount of genetic variation in a population, in a mutation-drift balance, is just a snapshot of the genes that are moving through it. If I were to go back to this diagram, and I were to put more genes into this process, and I were to ask you to go out and take a sample out of a population at any given time, you would take the sample at some time and you would tell me that's how many genes we have, that's how many are moving through. Okay? </p><p>Now the second possibility for a mechanism that will maintain genetic variation is a balance between mutation and selection. Mutation brings things into the population. Selection takes them out. So if we had a haploid population, with N individuals, and we have a mutation rate mu, we're getting Nmu new mutations each generation. The key idea is that if there is a mutation selection balance, then the number going in equals the number going out; that's what would keep this mechanism balancing the amount of genetic variation in the population. </p><p>And so if the mutant individuals have a lower fitness than the non-mutants, and if q is the frequency of the mutants, then selection is taking out NSq mutants per generation. And at equilibrium, with the number coming in equal to the number going out, the number coming in equals the number going out, and that gives us an equilibrium frequency of the mutation rate divided by the selection coefficient. It's a very simple result. </p><p>And if you do the same kind of thinking for a diploid population, you get that the equilibrium frequency will be the square root of the mutation rate, divided by selection for recessives, and the same as it is for haploids for dominance. Okay? So there are some examples of this. </p><p>There are rare human genetic diseases, such as phenylketonuria--that's the inability to metabolize phenylalanine. It has a frequency of about 1 in 200,000, in Caucasians and Chinese. It is probably in selection mutation balance. It's at low frequency but it's present in a population. People with it suffer a selective disadvantage. It keeps mutating and coming back in, and it keeps getting selected out. The result is balance, okay, and it's pretty rare. </p>
  <p>The third mechanism that will maintain selection in natural populations is a balance of selective forces; that is, where the heterozygote is better than either homozygote. And there is a classic, famous case, and it's always discussed in this context, and it's interesting that it's the one that's always discussed in this context, and the answer is it's been hard to find more. [Laughs] Okay? That's sickle cell anemia. </p><p>Now this is the normal heterozygote which is susceptible to malaria. The heterozygote is resistant to malaria, and the sickle cell homozygote is anemic and sick. And it sets up this kind of relative fitness. And, in fact, if--H here is actually going to be a negative number. Okay? So the fitness of the heterozygote is going to be higher than the fitness of either homozygote. And you can then set--the equilibrium frequency is going to be the one where P prime is equal to p; in other words, the frequency in the next generation is just the same as the frequency in this generation. </p><p>At what frequency does that happen? Well it happens when these little equations are satisfied. And the interesting thing, when you look at them, is that the selection coefficient has dropped out of them. The equilibrium frequency doesn't depend on the selection pressure, it depends on how frequently the gene is expressed in a heterozygote. So it depends really on the heterozygote advantage. </p><p>Now the real situation is more complicated than this. There are several such sickle alleles. They're changing frequency. The equilibrium assumption doesn't really apply out there in Nature, but it does give us a rough rule of thumb for how much to expect, and as soon as people who have sickle cell anemia move out of areas with malaria, it takes quite awhile for that allele to disappear from the population. </p><p>The fourth mechanism is a balance of selection forces, so that, for example, for A2, when A2 is 0, it has high fitness here, and as it increases in frequency its fitness drops, according to this equation. Now the frequencies of A1 are just reversed along this axis. A1 is 1.0 here, and it's 0 here. A1 has low frequency--has low fitness when it's at high frequency, and high fitness at low frequency. A2 has high fitness at low frequency; low fitness at high frequency. So both of them do better when they are rare. And I think that you can see intuitively from this diagram that at equilibrium they will stop changing when their fitnesses are exactly the same. </p><p>Now there are some interesting examples of this sort of thing. One is Ronald Fisher's classical argument on why 50:50 sex ratios are so common; why in many populations we see half females and half males. The deviations from that are interesting. This kind of thing happens with evolutionary stable strategies, and those are the solution to many problems within evolutionary game theory. They are also called Nash equilibria, under certain circumstances, and they are important in economics and political science as well. </p><p>And the tremendous amount of genetic variation in the immune system is thought to exist for reasons of frequency dependent selection; basically pathogen resistant genes gain advantage when they are rare, because when they're common, the pathogens evolve onto them. They are more or less sitting ducks; they're a stable evolutionary target. </p><p>But as they become more common and more and more pathogens evolve onto them, and those organisms get sicker and sicker, the ones that are rare have an advantage. And then as they start to increase in frequency, the same process occurs; the same process, it continues again, and after awhile you've got hundreds of genes, each of which is advantageous at low frequency, and none of which are advantageous at high frequency. </p><p>So this is a very important kind of mechanism maintaining genetic variation in natural populations, including our own. If we look at quantitative traits, such as birth weight--here's a classical example. This is for babies born in the United States in the 1950s and 1960s, and this is the percent mortality for babies of different weights. You can see that there's stabilizing selection that's operating to stabilize birth weight right at about 7 pounds, and there's variation around it. And you might wonder, why is there any variation around that? Why don't all babies have the optimal birth weight? It's such an important thing. And there are really two answers to that. </p><p>One is that there are evolutionary conflicts of interest between mother and infant, and father and mother, over how much should be invested in the infant, and these lead to some variation. And there's mutation selection balance. So that this is a trait which is probably determined by hundreds of genes, and at each of those genes mutations are coming into the population, and at each of those genes there is a mutation selection balance, and when you add that up, over hundreds of genes, you get quite a range of variation. Of course, some of this variation is also due to developmental effects of the environment; variations in the mother's diet and other parts of her physiological condition during pregnancy. </p><p>So to summarize. The origin and maintenance of genetic variation are key issues; mutations are the origin. Recombination has huge impact. There's a tremendous amount of genetic variation in natural populations. Remember that data from the HapMap Project on us, on humans, and that all of the differences that you have, in single nucleotide polymorphisms, from the person sitting next to you, and how you share them with people who have had a similar history since we came out of Africa. </p><p>We can explain the maintenance of this variation by various kinds of mechanisms, principally for balance between mutation and drift, between mutation and selection, and by some kind of balancing selection, either heterosis or frequency dependent selection. And we think that variation in many quantitative traits--human birth weight, human body size, athletic performance, lots of other things--is probably maintained by mutation selection balance, as well as by other factors. So next time I'm going to talk about the role of development in evolution.             </p><p>[end of transcript]</p><p>

  </p>
</div>
<p><a id="backToTop" href="#top">back to top</a></p>
        </div>