<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1" /><title>Principles of Evolution, Ecology and Behavior</title><body><div id="fileContents">
  <h2><strong> Principles of Evolution, Ecology and Behavior:</strong> Lecture 16 Transcript</h2>
  <table cellspacing="0" cellpadding="0" id="transcriptHeader" summary="transcript header">	<tbody>		<tr>			<td id="transcriptDate">February 18, 2009</td>			<td id="transcriptNav"><span id="forwardNav"><a href="javascript:history.go(-1);">&lt;&lt; back</a></span></td>		</tr>	</tbody></table></p><p><b>Professor Stephen Stearns:</b> Today we're going to talk about these phylogenetic trees that we've been discussing for the last couple of sessions; and this completes the three introductory lectures on methods that are used, or basic concepts that are used in macroevolution. So we began that with speciation, so that you can understand where the branches on the Tree of Life came from. And then we had a quick overview of how to construct a phylogenetic tree, so that you could see how the whole tree was put together. And now we're going to see what happens when you either lay these trees onto maps, or you put traits onto the trees, or you do both at the same time. This is basically comparative biology, in its modern sense. </p><p>The outline is first a bit about looking into time; using phylogenetic trees and looking into geographic history. Then we'll look at how we can map traits onto trees and draw some surprising conclusions. And then we will put trees together with traits and put them on maps and see what that tells us actually about evolutionary ecology of lizards on Caribbean islands. And then we'll end with a take-home message from comparative biology, which is that species are not independent samples, and that because they're not independent samples, we need special methods for trying to assess how frequently things evolve in the Tree of Life. And that will lead us into both looking at how seeds do in sun and shade, in related species, and the issue of whether you should be more faithful to your mate if you're going to be married to her for a long time. </p><p>Okay, well let's start with some of Godfrey Hewitt's work. And this has to do with what happened in Europe after the glaciers melted. Now just to remind you about what Europe looked like at the peak of the last Ice Age. The glacier came down out of Scandinavia and got down about into Northern Germany and Poland. The English Channel was dry because there was so much water that had been locked up in the continental ice sheath that the level of the earth's oceans dropped about 100 meters, and at that point there was a sub-glacial tundra that stretched essentially from Ireland all the way across France and through Russia, out into Siberia. </p><p>So there was pretty much--it was called the Mammoth Step. And at that point many of the animals that you now find in Northern Europe had retreated south, into glacial refugia. There was one in Spain, there was one in Italy, there was one in Greece and the Balkans, and in Asia Minor. And you can now take the mitochondrial DNA out of these various organisms that are plotted here, and you can reconstruct where they spent the last Ice Age, and how they got back into Northern Europe. </p><p>So, for example, most of the grasshoppers of Northern Europe came out of the Balkans, and the ones that spent that time in Spain never got over the Pyrenees. The hedgehogs managed to get over the Pyrenees and spread through France and Belgium and the Netherlands. However, about at that point they met the hedgehogs that were moving north from Italy, and there were a bunch of other hedgehogs that were coming up from the Balkans. </p><p>The bears, interestingly, managed to do just fine, getting out over the Pyrenees, and they moved right up into Scandinavia. And up in Northern Sweden they have met some bears probably that had been in the Ukraine and had come out from north of the Black Sea; and so forth. </p><p>So it's possible, using mitochondrial DNA, molecular phylogenies, to reconstruct the recent history of movements of animals across the planet, and trees, across the planet, and to understand why it is that there are certain places where we see hybrid zones. And in fact here are some hybrid zones, in Europe. These are places where you will frequently run into hybrids, and they are there because populations are coming back together that had been isolated in the Ice Ages and breeding with each other. </p><p>So, for example, this is one I know pretty well, because I went there for thirteen years. There's a spot in the Swiss Alps, in the eastern part of the Swiss Alps, just north of the Italian border, or just south of the Austrian border, right about here, where almost every flowering plant that you see is a hybrid. And the guidebooks are just lousy, they just are horrible. It's extremely difficult to identify what you're looking at. But when you see the big picture, you can understand it. So if you're interested in this kind of thing, this is a good paper to read, by Godfrey Hewitt, and the papers that have cited, that have cited that one; those are good sources. </p><p>Now what about humans? Well I'm going to show you first what's happened in about the last 10,000 years, and we're going to see that in Europe the agriculturists spread out from the Middle East and squeezed the Celts into the northwest. In Africa we're going to see the Bantu migration out of Cameroon, and how the Hottentots were squeezed into the southwest of Africa. And in Asia we'll see that agriculturists spread from both the Middle East and from China, and squeezed Siberians into the north. </p><p>So these things are laid out in a beautiful book by Cavalli Sforza, Paolo Menozzi, and another author, Alberto Piazza, I believe. And basically what they did was they tried to come up with a method of compressing a huge amount of genetic information onto a map, and they did it by taking gene frequencies, at hundreds of genes, and then compressing them, using statistical analysis, into a few factors, and then plotting those factors onto the map. </p><p>So what you can see here is basically the population differentiation of humans, in Europe, and you can see that there is kind of a wave that comes out of the Fertile Crescent and moves up to the north and to the west. And this tracks the agricultural expansion, which started about five or six thousand years ago, out of the Middle East. And you can see that the Celtic genes did get squeezed up into Ireland and England, and out into Brittany, and so forth. </p><p>So there are lots of neat details in this book. If you focus in on particular areas, you can see that there's a hotspot, right in Palermo, of Viking genes from the Viking occupation of Sicily, etcetera. Interesting stuff. </p><p>If you look at Africa, what you can see is that there has been an invasion of Africa by Caucasoid Northern Africans, and by sort of an Arab Nilotic expansion, coming down this way. The Bantu expansion out of Cameroon is what is coloring part of the continent pretty red. And, by the way, this migration got through here about a thousand years ago. So the Bantu migration down into East Africa is something that is relatively recent. </p><p>And if you've been reading about the war in the Congo, and the conflicts between the Hutus and the Tutsis, the Hutus are Bantu, and the Tutsis are Nilotic, and you can see where the Nilotic and the Bantu mix comes together here in the Great Lakes region of Central Africa. So this kind of map gives you some feel for the way things have moved. </p><p>By the way, there was a kingdom in Mali, centered around Timbuktu--or an empire--and this orange spot is a relic of that. So the history of human movement on the face of the globe is written in the genes and can, to a certain extent, still be recovered. This kind of study will no longer be possible after another few hundred years of jet travel and crossbreeding. My son, for example, is in a relationship with a woman from here. Another few generations of that and this map will not be reconstructable. </p><p>If we look at Asia, what we see is that there were nomads and agriculturists coming out of the area around the Middle East, and around the Black Sea and the Caspian Sea, that have pushed into Central Asia, and the Chinese agriculturalists have spread into Southern Asia. </p><p>But there's a very interesting hotspot of human biodiversity in southeastern Asia, going from India over about to Taiwan; and actually this is where the Polynesians came out of. The Polynesians left from Taiwan we think about 5000 years ago, and that's confirmed both in the language reconstruction and in the mitochondrial DNA. That just came out a few weeks ago. </p><p>So you can see that one of the themes of recent human history has basically been of the expansion of some groups at the expense of others, and that that was often a technology-driven thing, and often involved agriculture. </p><p>Now, I've shown you this before, and I just want to bring this back in again, at this point, to indicate what you can do with phylogenetic trees, and just to remind you. It is now possible to get information on single nucleotide polymorphisms at 650,000 different sites in the human genome, and this is a paper that did that for 928 unrelated individuals from 51 populations. So these are the 928 individuals out here. These names down here are the 51 populations, and the 650,000 different positions in the human genome are on the Y axis, all compressed together; so it's very hard to see any differentiation there. </p><p>And you can see that there is certainly a genetic signature across this; certain kinds of genomes in certain geographical areas. And if you then do the molecular phylogenetics on it, and construct the phylogenetic tree, you see that the oldest part of the modern human tree is centered in Africa. This, by the way, is the Classical view. This was a picture that could be drawn in 1995--and this is 2008; so this thirteen years later--and this tree largely confirms this picture. Okay? So you could lay this tree onto this map and come up with something that looks pretty much like this. </p><p>What you see here basically is we came out of Africa, we paused in the Middle East. Then various groups moved out of the Middle East. One group went into Europe, thought to be about 40,000 years ago; that's these guys. Other groups set out into Asia and spread out through Asia. And then out of the group that had settled basically in Eastern China and Japan and Korea, one group up here split off. Part of them--actually an early part of this branch--went down here, through Papua, New Guinea, into Australia, and another part went out into the New World. So phylogenetic methods can now be used to give us quite a bit of insight into our own history, as well as into the history of other plants and animals. </p><p>Now the Hawaiian Islands are an interesting test case. When we look at something like the human expansion across the globe, it's actually difficult to get precise markers for the times when they arrived in certain places. Archeology gives us some; sometimes we can recover fossil DNA from bones. But in the case of the Hawaiian Islands, at least on the scale of the last 5,000,000 years, we have very precise geological dates. </p><p>For example, we know that Kohala Mountain, the oldest rock on Kohala Mountain is 430,000 years old, and that the oldest rock on Kauai is 5.1 million years old; and that's because the islands are made over a hotspot here, and carried on a plate up in that direction. And you can actually lay down, on this plate, how long ago it was that that island was actually sitting down here. And that's nice, because when we then do phylogenetics and we start putting phylogenetic trees onto this map, it gives us some feel for when in time those different branch points might have been. </p><p>And that has been done for a number of groups. I'm sure there's now more information. This is about five years old here. And these are three different ways that spiders and some other arthropods and some insects speciated over the last 5,000,000 years in Hawaii. Interestingly, you can see that they all moved down from Kauai onto the younger islands. So they were going from older islands onto younger islands, and they just kept hopping. </p><p>And, by the way, if you just continue this island chain up to where it dives into Siberia, there were islands there 350,000,000 years ago that are now getting subducted under Kamchatka, and there are things that are one or two-thousand miles up to the northwest that were once high islands above water. So this hopping movement could have been going on for quite awhile. And, in fact, we think that some of the things that we now see in Hawaii got there about 20 or 30,000,000 years ago, before Kauai came above water. So it's an old process. </p><p>At any rate, in some cases things simply moved from one island to the other, and then every time they got onto a new island they speciated. In other cases there were four or five species from Kauai that moved down to Oahu, and then several of them sorted out on Oahu and speciated on Oahu, and four or five of them moved down to Maui Nui, and so forth, and then a lot of them speciated on Maui Nui and moved over to the Big Island; and that process went on. </p><p>Another process is one where you have a lot of species on Kauai. One of them moves to Oahu. You get a lot on Oahu, but only one then is the ancestor that goes on to Maui Nui, and you get a lot there, and then one goes onto the Big Island, and then you get a species complex on each of the five volcanoes on the Big Island, that are coming from that one ancestor. So all of these processes can actually be seen written in the genes, and this is all resulting basically from sequencing, either nuclear or mitochondrial DNA. </p><p>There are other kinds of questions that you can answer. And this is one that was answered by Anne Yoder, who was on the faculty here. She now is the head of the Duke Primate Center. And Anne has specialized in the mammals that live on Madagascar. And on Madagascar you find a local radiation of things that look kind of like civet cats or mongooses. And the question was, did they come over from Africa separately, or did they all speciate on Madagascar? </p><p>And Anne was able to reconstruct the phylogeny of this group of animals well enough so she could lay this tree onto this map and determine that in fact these guys are actually all relatives of mongooses, and mongooses are close relatives of hyenas, and those things have a sister group of civets, and those things have a sister group of cats. And they got across right here. </p><p>Now Madagascar split off from Africa about 65,000,000 years ago. It's part of the Tectonic breakup that led to India splitting off, skitting across the Indian Ocean, crashing into Asia, and raising the Himalayas. And so you might wonder, well how the heck does something like this cross a strait which is now more than 200 miles wide? </p><p>Well have any of you read Rudyard Kipling's story, <i>How the Elephant Got His Trunk</i>? "Down by the grey-green, greasy Limpopo River, all set about with fever trees"--the elephant child looked into the water, and a crocodile grabbed his nose and pulled it out, and that was how the elephant got his trunk. </p><p>Well the grey-green, greasy Limpopo River is right here. And in flood stage, if you go out on the Limpopo River, there are large rafts of trees and vegetation being carried down it, and in a good storm you can put a mongoose on a raft of vegetation and get it out across that strait; so it rafted across, and it probably came out of the grey-green, greasy Limpopo River. Okay? So that's right here. </p><p>Now what about issues--I'm running through a series of issues that can be resolved using comparative methods in phylogentic trees. And this represents one of the phylogenetic surprises. There are parasitoid wasps which can be either ectoparasites or endoparasites. The ectoparasites lay their eggs on the outsides of caterpillars, and the eggs hatch and make little baby wasps that crawl around the outside of the caterpillar, and they eat it from the outside. </p><p>And they have relatives who are endoparasites. The Ichneumonids are like that. Here's a nice Ichneumonid-like wasp. You can see her long ovipositor. And here she is on an insect larva, and she is injecting an egg into it. And the ancestors turn out to have been wasps that did this. </p><p>It had been thought that if you just look at it mechanically, it would be easier to start the evolutionary process off with the wasp just flying around and laying its egg. Right? But in this particular radiation--that may have been the case far in the past; somewhere off the slide out here there may have been an ectoparasitic wasp that was an ancestor. But it turns out that all of these things are endoparasites, and the ectoparasites evolved within that, and then within the ectoparasites you had a reversal again and got some endoparasites out of it. </p><p>So if you lay the traits onto the tree, and you are confident of your tree, you can reconstruct the history of an important trait like this; and that's a nice kind of insight to be able to have. </p><p>Okay, now I would like to discuss the Anolis lizards. And this is something that a large group of scientists, currently centered at Harvard but with branches in Seattle and in St. Louis and other places, have been working on for about the last fifty years. And they study Anolis lizards because these are prominent, easily observed; you can get a pretty good sample size fairly quickly. And they've done fascinating things. So the Anolis lizards have had a big radiation on islands in the Caribbean, and they have made what are called ecomorphs. </p><p>Now these ecomorphs can be grouped by appearance. So if you look at them and you see where they're living and what kinds of grasping appendages they have, basically what the phenotype looks like and how they behave, you can come up with things that live on the crowns of trees; out on twigs; down in the ground in grass and in bushes; between the trunk and the crown; only on the trunk; or down on the ground and going up on the trunk. Okay? So there are six ecomorphs. </p>
  <p>And you find them again and again. You find them on many islands in the Caribbean. And it's really tragic that in the middle of winter, up here in New England, you have to be continually flying down to the Caribbean to increase your sample size, okay? And doing that for twenty or thirty years, through many generations of graduate students. </p><p>Well, after DNA sequencing came along, you could do their phylogeny. And look just--it's hard to see because the reproduction isn't very good on the slides--but just look at the difference in the color pattern. Okay? So, for example, this is the trunk crown's space. Well, it turns out to pop up here, and here, and here, and here: one, two, three, four times, independently. Or you could take something like the twig form. Here's a twig form, here's a twig form, and here's a bunch of twig forms. </p><p>Now what's going on here basically is you're having the independent evolution and the convergence on different islands of these different ecomorphs, among these lizards. And if you take that down and you break it down by island, what you can see is that on Cuba the trunk form, the crown trunk form was ancestral. On Hispaniola, the twig crown form--or no, this is trunk crown and this is crown giant; this is a crown giant form, was the ancestral form. </p><p>So these are inferences about what first got onto that island and what first got out to the other island. And then these other things all evolved from that. And what you see from that basically is that it doesn't matter which form of lizard you first throw onto an island; all of the other ones are going to evolve from it. And these are all different species. And we're talking about things that range in size from about that big, up to about that big, and that are differentiated at the level of genera. </p><p>So this is really major evolution, which is being repeated again and again, across the Caribbean, and generating essentially--what's going on here is that essentially the same ecological community is being generated again and again, independent of which kind of species founded that group. Okay? That was highly unexpected; people didn't think this would happen. So this means that you're getting the same ecomorphs from different ancestral states, and that means you've got convergence. </p><p>Okay, now for an insight from a guy named Joe Felsenstein. And if you want to go back to the original paper, it's in 1985. So this is one of Joe's many contributions to phylogenetics. If you just look at this picture that I showed you the other day, you can see that the red trait evolved in the ancestor of both B and C. Okay? So that means that B and C share a trait. However, in evolutionary terms, it only evolved once. </p><p>Now at this point, the reason that red increased in frequency might have been that it was adaptive. Things that had the red state had a fitness advantage. So microevolution was driving it at that point. But then everybody inherited it, and it is not an adaptation to the difference between whatever environments B and C now live in. So if you were to look at B and C now, and you saw that they shared some trait, you wouldn't really know why that was there, until you could get a much, much larger sample size; because you just have a sample size of one, when you're looking at that trait. Okay? </p><p>So how do you deal with that problem? Well Joe came up with what he called the method of independent contrast. And in this context a contrast is the difference of the value, the mean value of a trait in one species and its value in another species. And if you look up just at the tips of this phylogenetic tree and you take the differences across the closest related sister pairs, at the tops of the tree, you generate these contrasts. You get X2 minus X1; X4 minus X3; and so forth. </p><p>Well the important thing about the contrasts is this. The difference that evolved, after this point on the tree, is independent of the difference that evolved after this point on the tree. You've taken out whatever was there because of the common ancestor. So whatever was going on over in this part of the tree is biologically separated and now statistically separated by this method, from whatever was going on in any other part of the tree. </p><p>So this actually is a method of getting the correct sample size, off a phylogenetic tree. And that's very important in statistics, because if you have the wrong sample size, all your statistical tests will be wrong. So this was important for the mental health of people who were doing statistics on phylogenetic trees. </p><p>So let's take a look at some approaches that are kind of like that. So this is from a guy named Peter Grubb. Peter was the president of the Ecological Society in the UK. He's a botanist at Cambridge. And what he's done here is plot the log of seed mass of light-demanding seeds, against the log of seed mass of shade- tolerant seeds. And his question was this: Do plants living in shade produce larger seeds than plants living in sun? And he wanted to do a phylogenetically controlled comparison. </p><p>So what he's doing here is he's taking essentially the value within a genus, or within a family--the mean value of species within a genus or the mean value of genera within families--for related trees, some of which live in open areas and demand light for their germination, and others of which have seeds that can germinate and survive in the shade. So the open circles are comparing genera within families, and the closed circles are comparing species within genera. And what you see here basically is this. </p><p>Plants living in shade do produce larger seeds than plants living in sun, but you only see it in comparison of genera within families; you don't see it in comparison of species within genera. Okay? So the ones that need light and the ones that need shade have just about exactly the same seed size, if you are looking at species within genera. But if you then give them longer to evolve, you go further out on the phylogenetic tree, you compare things between families, where that contrast is possible, then you start to see them moving off the one-to-one line, and the ones that are shade-tolerant have seeds which are falling quite a bit--not all; this is an exception--but quite a bit above the line. </p><p>So this is a way not only to answer that kind of question, using the comparative method, but also to get an estimate on how long it takes. It takes a long time to generate that difference, because you only see it at a higher level on the tree. </p><p>Now what about the albatrosses and their relatives? These are the Procellariiformes, the Tubenoses. You can see the Tubenose right here--and that's a Wandering Albatross; and you can see the Tubenose right here on this Petrel. And these things have totally different life histories. Okay? </p><p>The Wandering Albatross, which has a wingspread of between twelve and thirteen feet, and is probably the heaviest flying bird, it lives on islands in the Southern Ocean. And it mates, usually starting at about the age of twelve or fifteen or so, and it usually will produce an offspring every other year or so, for about thirty or forty years. They mate for life; they're monogamous. And they have very precise homing behavior to their chicks. They lay their eggs on places like South Georgia, Macquarie Island, places in the Southern Ocean. </p><p>And some French biologists put a radio collar on one of these Wandering Albatrosses and tracked the mother as she took off to go get lunch, for baby. And she flew north, from South Georgia, and she peeled off towards Australia and flew up the West Coast of Australia, and came back across the Indian Ocean and down the East Coast of Africa, and back down to South Georgia a month later; at which point baby, by that point, was extremely hungry, got a lunch of rotten squid. </p><p>And this wide foraging means that they're only going to be raising one child every two years or so; and there are all kinds of adaptations in the infant's physiology to deal with this irregular feeding. </p><p>A Storm-Petrel is a lot smaller. It forages much closer to shore. It is not so faithful to its mate. These things are related. Okay? So if you look within this family, can you ask the question about whether or not a long life really is a situation that promotes mate fidelity? The argument there is basically that if you're going to have a short life, there isn't going to be enough time to pick up the advantages that you would get from knowing a particular mate, and adapting behavior to that particular mate, and learning exactly how to be a good parent with that particular mate, rather than with some other. </p><p>And if you look across the Procellariiformes; so this would be the Wandering Albatross up here, and this would be a Petrel down here, and you've got some other things in the middle--these are all separate species now; the dots here are all species. And we have here a study in which--you are looking at independent contrasts now, and we're plotting the independent contrasts. So this is the deviation in adult life expectancy from an overall mean, for the whole group, and this is the deviation in mate fidelity from an overall mean. </p><p>So the ones that tend to live a long time are more faithful to their mates, and the ones that tend to live, have a short life, are much less faithful to their mates and switch mates. And that's interesting because in fact these guys, the ones that reproduce many times, have much more opportunity. They have thirty or forty years of reproduction. They could go out and they could get divorced and pair again several times. But they don't, they stick together. And the functional reasons for that are things that are not really terribly well understood. </p><p><b>Student:</b> What's negative mate fidelity? </p><p><b>Professor Stephen Stearns:</b> What?</p><p><b>Student:</b> Negative mate fidelity.</p><p><b>Professor Stephen Stearns:</b> Negative mate fidelity is just a statistical thing. They are taking the overall--they've measured mate fidelity on some scale, they've taken an average for the whole group, and then they are asking, how far does this species deviate from the average? So if it's below average, it gets a negative number, and if it's above average it gets a positive number. Okay? </p><p>Okay, so now I want to ask you a question that comes right out of molecular phylogenetics, and I want to see whether or not you can actually put together some information that you've now gotten from different lectures. So this is going to require you to piece together phylogenetics and genetic drift. So this is the figure from the Becky Cann/Allan Wilson paper that came out now twenty years ago. It's basically on human mitochondrial evolution. </p><p>And what it showed was that all human mitochondria appear to be derived from an ancestor, all of whose closest relatives now--the close relatives now are all out at the tip of the tree--lived in Africa. You don't start picking up non-African members of this tree until you get out to this point, and then you can see that by the time you get way out on the tree, that most of these now are non-Africans. </p><p>So assertion number one out of this is, hey, human mitochondria show that we came out of Africa. Well we now know from that paper I showed you more recently, from SNP polymorphisms, that this is an extremely well supported thing, and that we see it in the nuclear genes as well. </p><p>But the claim here was an interesting one. It said there was one woman, living in Africa, about 220,000 years ago, from whom all other mitochondria in all humans on the planet are descended, and so they gave her the name Mitochondrial Eve; that's observation number one. </p><p>Observation number two--and this now comes from an immunobiology group in Germany--and basically it has to do with how old are the polymorphisms in our MHC genes? And these are things that have been selected, probably through frequency dependent selection, in co-evolution with diseases. </p><p>And the observation is this. Say we have two MHC genes that have resulted from a gene duplication--and that would be this one here and this one here--and at each of those genes there is a polymorphism, so that we have different alleles at that locus, for each of those two genes. Who are those alleles most closely related to? </p><p>Well it turns out that Allele 1 in humans is most closely related to Allele 1 in chimps, and Allele 2 in humans is most closely related to Allele 2 in chimps. In other words, the closest relatives of the alleles are not in this species; they're in another species. The only possible way that that could have occurred is if the polymorphism originated before the speciation event--okay?--so that you had ancestral species here, and this polymorphism originated, and you got this one coming down from the ancestral species into the chimp, and into the human; and this one coming down from the ancestral species into the chimp and into the human. </p><p>Now, on the one hand we have the claim all the mitochondria came out of one person. On the other hand we have evidence that there are trans-specific polymorphisms. And if you take this point in time and you put it on this tree, it's about here. I now want you to talk to each other for a minute, and then we'll see if you can come up with the explanation for how those two observations are consistent with each other. So just turn to your partner and figure it out.</p>
  <p>[Students confer with one another]</p><p><b>Professor Stephen Stearns:</b> Silence descends. Enlightenment has been reached. Okay, can anybody tell me why they are not surprised that all of the mitochondria came from one female? Why are you not surprised?</p><p><b>Student:</b> Because they're going to come down through meiosis. It comes down&hellip;</p><p><b>Professor Stephen Stearns:</b> It's asexually inherited. How many females do you think there were in the African population 220,000 years ago? Do you think the second observation tells us anything about that, the trans-specific polymorphism? </p><p>What happens when a population is really small? You get genetic drift. If a population were as small as one female, it would've been impossible to maintain this trans-specific polymorphism. People have done simulations to find out what is the average size of a population that would, over a period of 5,000,000 years, maintain the amount of trans-specific polymorphism that we see in our MHC complex; in other words, the amount of genes that we share with chimpanzees, where our alleles are more closely related to the chimp's alleles than they are to the other human alleles? </p><p>And the answer is the minimum size is about 10,000. In other words, we have good genetic information that tells us that the smallest the human population ever has been, over the last several hundred-thousand years, actually over the last 5,000,000 years, since we shared ancestors with chimps, is about 10,000. Okay? Given that, now tell me, are you surprised that we can trace all the mitochondria, in all the females on the planet, back to one woman, living in East Africa about 220,000 years ago? And if you're not surprised, I want to know why. Yes?</p>
  <p><b>Student:</b> Well you were talking before about how recessive genes spread out in a small population. So if she had enough children over enough generations, you'd have enough of a population [inaudible]. </p><p><b>Professor Stephen Stearns:</b> Okay, it's possible that in fact she had a particularly advantageous mitochondrion, and that it then got selected and it fixed, and then everything would go back to her. That's correct. And that could've been done in a larger population. However, it could also have been done with drift, and it happened so long ago that we can't really tell whether it was selection or drift that gave that one woman the advantage. </p><p>By the way, the same thing has been done for the Y chromosome. Okay, the Y chromosome is also asexually inherited, and the estimate on the Y chromosome is roughly also about 200,000 years ago, also in East Africa. And the fact that the mitochondrion and the Y chromosome both converged to a common ancestor, at about the same age, might suggest that drift is more likely than selection to explain it. </p>
  <p>But basically what's going on, if you think about it, is that in any process like that, if you go back far enough in time, it will always converge on a single common ancestor. Okay? Now the next thing I'd like to tell you is that there was a controversy about when that happened; and the controversy on dating that point of convergence is all about how long has it been since we split with chimps? Because that turns out to be the baseline that gives us an estimate of how rapidly evolution is going on, molecular evolution is going on, in the human clade. Well when you apply that criterion to what the confidence limits on the estimate are, hey, it was anywhere from last year to about a million years ago. [Chuckles] </p><p>So the confidence limits are lousy. The observation, however, that that person was in Africa is pretty solid and, as I say, is now confirmed by the nuclear genes, the SNPs that I showed you earlier in the lecture. So the point of this exercise is that when you see results like this, when you see some claim of Mitochondrial Eve, or Y Chromosome Adam--which are both out there in the literature; just go on Web of Science, type Mitochondrial Eve, you'll pick up a lot of controversy about this. </p><p>I want you to realize that (a) we should never be surprised if particular mitochondria or particular chromosomes converge at some point in time--it looks like they were just in one individual. That's just because of the way that branching processes work, and that will be going on all the time, and it's happened over and over and over again. So it's no surprise that it goes back to one person. </p><p>The other point that I would like to bring out is that if you contrast different kinds of historical evidence, you can often gain enlightenment by seeing that there is a puzzle that needs to be solved. And in this case the puzzle basically is that this tells us something about population size, and this tells us very little about population size. </p><p>This process, convergence back to a single ancestor from the mitochondria, it's not entirely independent of population size--it'll take longer in a big population than it would in a small population--but it doesn't give us an estimate of how big the population had to be. Whereas the trans-specific polymorphism could only have been maintained, even with strong selection, in a population that was larger than about 10,000 individuals; and that can be done with computer simulations. </p><p>So these are different forms of enlightenment into the history of genes in a particular clade; a history that happens to matter to us quite a bit. And we can gain that by looking carefully at phylogenetic trees. These are actually both phylogenetic trees. This tree here is just laid on its side, and you'll find that this is commoner and commoner practice these days. </p><p>If you look at the cover of <i>Nature</i>, or <i>Science</i>, or the--actually the one I remember is in the <i>New York Times </i>Science Section for Darwin's birthday, on February 12<sup>th</sup>. That week the Science Section had a phylogenetic tree covering Darwin's face, and it had thousands of species on it, and they're laid out this way, just so that you can fit the species onto a piece of paper. So time kind of wraps around, in the way it's presented. </p><p>Okay, so to summarize this part of our exploration of macroevolution. These molecular methods allow us to reconstruct geographic movements, as well as phylogenies. And we saw that in the hedgehogs going north from Spain and the Balkans, and we saw it in the humans moving out of Africa, and we saw it in lots of things. </p><p>We see that our own migrations have left genetic traces on all the continents. And there's an imprecise map that's suggestive between the genetic geography and the linguistic geography. Greek genes go with Greek family names, from the boot of Italy, up to about Rome, and then stop; that kind of thing. So even in the last two or three-thousand years, you can see that family names and genes have been inherited in similar ways. </p><p>We can use these methods to determine which trait states were ancestral and which are derived. And that was particularly interesting in the case of the parasitic wasp, whether it was an ectoparasite or an endoparasite, because it changed received opinion about fundamental biology. And this method that Joe Felsenstein worked out for independent contrast is something that will control for common ancestry, and it can reveal the correlated changes in two or more traits that have taken place since branches in the tree. So it can be used as a fairly powerful method to explore hypotheses in behavioral ecology, evolutionary ecology, and ethology. </p><p>So next time we're going to start talking about key events in the history of life; it's the first of three ways to look at the history of life.     </p><p>[end of transcript]</p><p>

  </p>
</div>
<p><a id="backToTop" href="#top">back to top</a></p>
        </div>